ambry-frontend/src/main/java/com.github.ambry.frontend/AmbryBlobStorageService.java

```diff
@@ -208,6 +224,17 @@ public void handlePost(RestRequest restRequest, RestResponseChannel restResponse
       logger.trace("Handling POST request - {}", restRequest.getUri());
       checkAvailable();
       long propsBuildStartTime = System.currentTimeMillis();
+      accountAndContainerSanityCheck(restRequest);
+      // check if the request is old (with no accountName nor containerName) or new (otherwise)
```

Comment:
Have wrapped all account/container related logic in one method called `injectAccountAndContainerForPutRequest`